x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -142,6 +143,14 @@ export const links: LinkItem = {
           defaultMessage: 'Exception lists',
         }),
       ],
+      links: [
+        {
+          id: SecurityPageName.sharedExceptionListDetails,
+          title: SHARED_LIST_DETAILS,
+          path: '',
+          skipUrlState: true,
```

Comment:
If you don't want to display the timeline footer there you should add: `hideTimeline: true`

Reply:
I tried with and without `hideTimeline: true` it doesn't show the Timeline, so I don't really know why we need it 🤔

And even when I add this flag, the below test in `use_show_timeline.test` fails 😫,
```
 it('hides timeline for sub blacklist routes', async () => {
      mockUseLocation.mockReturnValueOnce({ pathname: '/administration/policy' });
      await act(async () => {
        const { result, waitForNextUpdate } = renderHook(() => useShowTimeline());
        await waitForNextUpdate();
        const showTimeline = result.current;
        expect(showTimeline).toEqual([false]);
      });
    });
 ```

Reply:
`hideTimeline` is only used by the new navigation. So, make sure you enable it for testing.

To enable the new navigation go to advanced settings (`/app/management/kibana/settings`) and search for `securitySolution:enableGroupedNav`.

Reply:
Thanks @machadoum I have the flag enabled, and still when I add `hideTimeline: false` it doesn't show the Timeline still :(, the point is when I add the flag to false the above test fails :(